.gitignore

```diff
@@ -20,6 +20,7 @@ latex/
 *.pdf
 *.tag
 .DS_Store
+locm3.sublime-workspace
```

Comment:
According to the documentation here (https://www.sublimetext.com/docs/3/projects.html), .sublime-project should be in version control (since it contains things like paths and tab size), and .sublime-workspace shouldn't (since it contains things like which files are currently open in the editor).